tests/client_tester.py

```diff
@@ -15,54 +15,55 @@
 
     # CHANGE FLAGS HERE TO TEST SPECIFIC FUNCTION TYPES
     test_creates = False
-    test_updates = False
+    test_updates = False # To test updates, must also test creates
     test_gets = True
     test_deletes = False
 
     # CHANGE FLAG TO PRINT ALL OBJECTS THAT FUNCTIONS INTERACT WITH
     print_objects = True
 
     objects_to_test = [ # CHANGE TO TEST DESIRED STREAMS 
-        'items',  # GET - DONE | CREATE - DONE | UPDATE - 
-        'categories',  # GET - DONE | CREATE - DONE | UPDATE - 
-        'discounts',  # GET - DONE | CREATE - DONE | UPDATE - 
-        'taxes',  # GET - DONE | CREATE - DONE | UPDATE - 
-        # 'employees',  # GET - DONE | CREATE -  | UPDATE - 
-        'locations',  # GET - DONE | CREATE - DONE | UPDATE - 
+        # 'items',  # GET - DONE | CREATE - DONE | UPDATE - DONE
+        # 'categories',  # GET - DONE | CREATE - DONE | UPDATE - DONE
+        # 'discounts',  # GET - DONE | CREATE - DONE | UPDATE - DONE
+        # 'taxes',  # GET - DONE | CREATE - DONE | UPDATE - DONE
+        'employees',  # GET - DONE | CREATE -  | UPDATE - 
+        # 'locations',  # GET - DONE | CREATE - DONE | UPDATE - DONE
```

Comment:
Why comment the other things?

Reply:
I was just testing employees last. This isn't executed in circle, so the state of this list isn't too important.

Reply:
Ok